src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -9013,13 +9013,15 @@ paths:
                   description: RBD image name
                   type: string
                 rbd_image_size:
-                  default: 1024
                   description: RBD image size
```

Comment:
We should maintain this description into the legacy param as well.

Reply:
I'd expect the legacy parameter to be dropped from the API reference, even if it continues to be accepted.  Otherwise please make it clear which one is legacy ;)

It would be nice to also explicitly mention that the units are same (am I right?).  Before I checked the history, seeing that one of the two defaults to 1024 made me think that the reason for the original incompatible change was to move from megabytes to bytes.  `git log` revealed that the reason was just naming, but if the value for both is in bytes the default to two 512-byte sectors makes no sense, so I'm still confused.

Reply:
The main issue here is that the deprecation happened "accidentally": a previous cosmetic PR renamed `size` to  `rbd_image_size`, and that broke compatibility with existing NVMe API clients (e.g.: IBM vSphere). We're recovering it to ensure backward compatibility.

Reply:
oh correct! good catch!

Reply:
@idryomov yes its very much cosmeti , we are adapting the changes to support backward compatibility , see this #61698

Reply:
> see this #61698

This doesn't explain the bogus default for the old parameter ;)

Reply:
Okay thats something I am also now thinking why is that so ? And checking more found this https://github.com/ceph/ceph-nvmeof/issues/631 the support is in order of MiBs and this is something either we missed/not followed.

Reply:
https://tracker.ceph.com/issues/70106